package.json

```diff
@@ -36,13 +36,13 @@
   "dependencies": {
     "@auth0/auth0-react": "^1.1.0",
     "@opentripplanner/base-map": "^2.0.0",
-    "@opentripplanner/core-utils": "^5.0.2",
+    "@opentripplanner/core-utils": "^6.0.0",
     "@opentripplanner/endpoints-overlay": "^1.4.1",
     "@opentripplanner/from-to-location-picker": "^2.1.2",
     "@opentripplanner/geocoder": "^1.2.1",
     "@opentripplanner/humanize-distance": "^1.2.0",
     "@opentripplanner/icons": "^1.2.2",
-    "@opentripplanner/itinerary-body": "^3.0.5",
+    "@opentripplanner/itinerary-body": "^4.0.0",
```

Comment:
Hmm see this? Not a big deal, but for next time, try to limit breaking changes commits to just the affected packages (should have been just for core-utils, not for itinerary or transitive).

Reply:
Yeah, I noticed that too late... I think it was because I had to break apart the PR after the fact with cherry picks and reverts.